src/styles/index.scss

```diff
@@ -1,3 +1,138 @@
+$button-color:#00acdc;
+$second-text:#616070;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background: rgb(213, 213, 209);
+}
+
+$star-size: 16px;
+
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: $star-size;
+    height: $star-size;
+    flex-shrink: 0;
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+    background-size: auto;
+
+    &:not(:last-child) {
+      margin-right: $star-size / 4;
+    }
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} &__star:nth-child(-n + #{$i}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
+
+.product-card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  position: relative;
+  background: #fff;
+  margin: 0;
+  color: #060b35;
```

Comment:
add `padding: 32px 16px 16px ` to your card to avoid using extra wrappers (mentioned it above - `.product-card__body` is redundant)